src/main/java/org/opensearch/knn/index/KNNSettings.java

```diff
@@ -409,11 +397,7 @@ public void onResponse(ClusterUpdateSettingsResponse clusterUpdateSettingsRespon
 
             @Override
             public void onFailure(Exception e) {
-                logger.info(
-                    "Exception while updating circuit breaker setting {} to {}",
-                    clusterUpdateSettingsRequest.persistentSettings(),
-                    e.getMessage()
-                );
+                logger.info("Exception while updating setting {} to {}", clusterUpdateSettingsRequest.persistentSettings(), e.getMessage());
```

Comment:
No I dont think so, I will update.

Reply:
I am pretty sure it will. you are doing e.getMessage(), which will print the message only. 

The way to fix this is

```
logger.info("Exception while updating setting {}", clusterUpdateSettingsRequest.persistentSettings(), e);
```